wp-seo-main.php

```diff
@@ -308,91 +309,10 @@ function wpseo_frontend_head_init() {
 }
 
 /**
- * Register the promotion class for our GlotPress instance
- *
- * @link https://github.com/Yoast/i18n-module
- *
- * @return yoast_i18n
- */
-function register_i18n_promo_class() {
```

Comment:
The 'neat' way would be for the function to remain, but with a deprecation notice and re-route to the new location/method...